TRANSLATING.md

```diff
@@ -0,0 +1,105 @@
+2. How long are you going to stick around in bot development?
+   grammY is young.
+   It evolves quickly and there are many articles that get changed often.
+   The translations must be kept in sync with the English original.
+   Outdated and wrong docs are much worse than no docs.
+   Many people only come here for a few months (and that's fine!), but if you translate everything and then stop developing bots and stop updating the articles, we'll have to remove that language again.
+   Then it was a waste of time for you.
+   So if you don't want to commit long-term, don't start.
+3. How many hours per week can you spend in the beginning?
+   It's understandable that a first PR doesn't contain translations for all pages immediately.
+   However, if you can only work on it rarely, it's going to take a very long time until you're done.
+   We don't want to end up with 15 languages and none of them is done for months.
+   If you can't set aside enough free time to get this done, don't start.
+
+Still eager to join us?
+Great!
+Here is how we work:
+
+## The Translation Workflow
+
+Translating the grammY docs into a language happens in two stages: the Active Stage and the Maintenance Stage.
+
+When a language is first started, no translations are available.
+Hence, the complete website must be translated actively (Active Stage).
+This requires an insane amount of work and a fair bit of dedication.
+Do not underestimate the time you will need for this.
+Respect to everyone who manages to complete the work!
+
+The language will then enter Maintenance Stage.
+From here on, all that needs to be done is to keep notifications on for new pull requests, and sync the translations whenever an article gets updated.
+
+Here is some more information about how each stage works.
+
+### Active Stage
+
+A translation to another language must be 100 % complete before it gets published.
+We do not accept partial translations.
+All menus, links, articles, comments, and everything translatable must be localised before the changes can finally go live on the website.
+Please open a translation tracking issue so everyone can follow the progress.
+It can look like https://github.com/grammyjs/website/issues/55.
+
+It would be very painful to do everything in a single pull request, so we developed a simple branching model which will make this easy.
+
+Once you start working on a new language, we will give push permission to this repository.
+You can then create a new branch for the translations, e.g. `feat/spanish-translations`.
+This is called the _translation branch_ of your language.
+
+For every page you translate, create a new branch, perform the translations, and open a pull request.
+Make sure to open a pull request against your translation branch, and not against `main`.
+
+A different person who can speak your language can now review the pull request.
+Make sure to reiterate on the translations, in case you have ideas to improve it.
```

Comment:
```suggestion
Make sure to reiterate on the translations, in case you have ideas to improve it.
The GitHub bot will provide you with a website preview of your pull request, so you can review a fully rendered version of how your changes will look, which will enable you to catch all mistakes.
```

Reply:
Actually, I should say sorry because I should've done this.